Essentials/src/main/java/com/earth2me/essentials/commands/Commandsettpr.java

```diff
@@ -17,22 +19,21 @@ public Commandsettpr() {
 
     @Override
     protected void run(final Server server, final User user, final String commandLabel, final String[] args) throws Exception {
-        if (args.length == 0) {
+        if (args.length < 2) {
             throw new NotEnoughArgumentsException();
         }
-
         final RandomTeleport randomTeleport = ess.getRandomTeleport();
-        randomTeleport.getCachedLocations().clear();
-        if ("center".equalsIgnoreCase(args[0])) {
-            randomTeleport.setCenter(user.getLocation());
+        randomTeleport.reloadConfig();
```

Comment:
This invokes an I/O operation. Is there a reason why it's ran let alone before any arguments are even validated?